apps/dot_ext/models.py

```diff
@@ -116,6 +120,27 @@ def get_expires_in(self, client_id, user_id):
             return None
 
 
+class Approval(models.Model):
```

Comment:
It'd be great if we could split the model def and related migration into a separate branch and PR. I think we'll want to run the migration before putting the functional changes in place to avoid trouble.

Reply:
The deployment should run migration first. But I agree it might be safest to deploy the model and the migration first and then a second PR to implement code that uses the new model.

Let's test in DEV as one PR and if it deploys smoothly we can go with 1 PR.